ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/min-permissions/provision/cucushift-installer-rehearse-gcp-ipi-min-permissions-provision-chain.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/min-permissions/provision/cucushift-installer-rehearse-gcp-ipi-min-permissions-provision-chain.yaml",
```

Comment:
Suggest to rename `min-permissions` to `minimal-permission` to keep consistent path like aws.